internal/exec/process.go

```diff
@@ -54,12 +55,13 @@ type ProcAttr struct {
 }
 
 type SysProcAttr struct {
-	HideWindow        bool
-	CmdLine           string // used if non-empty, else the windows command line is built by escaping the arguments passed to StartProcess
-	CreationFlags     uint32
-	Token             syscall.Token               // if set, runs new process in the security context represented by the token
-	ProcessAttributes *syscall.SecurityAttributes // if set, applies these security attributes as the descriptor for the new process
-	ThreadAttributes  *syscall.SecurityAttributes // if set, applies these security attributes as the descriptor for the main thread of the new process
+	HideWindow         bool
+	CmdLine            string // used if non-empty, else the windows command line is built by escaping the arguments passed to StartProcess
+	CreationFlags      uint32
+	Token              syscall.Token               // if set, runs new process in the security context represented by the token
+	ProcessAttributes  *syscall.SecurityAttributes // if set, applies these security attributes as the descriptor for the new process
+	ThreadAttributes   *syscall.SecurityAttributes // if set, applies these security attributes as the descriptor for the main thread of the new process
+	ProcThreadAttrList *winapi.PROC_THREAD_ATTRIBUTE_LIST
```

Comment:
could you add a comment here like the above?

Reply:
Can do